src/patternfly/components/Card/card.scss

```diff
@@ -29,6 +29,11 @@
   --#{$card}__header--m-wrap--RowGap: var(--pf-t--global--spacer--sm);
   --#{$card}__header--m-wrap--ColumnGap: var(--pf-t--global--spacer--md);
 
+  // Description
+  --#{$card}__description--FontSize: var(--pf-t--global--font--size--body--sm);
+  --#{$card}__description--Color: var(--pf-t--global--text--color--subtle);
```

Comment:
Hmm... did you mean to add these styles in? Right now they aren't applied so the description element looks the same as a regular card body element.

Assuming you do, you'll also want to add an entry in the block below for the description to set it to a disabled text color on a disabled card. https://github.com/dlabaj/patternfly/blob/31c10253b87d0ac9c3bd11c2b883b10134064c83/src/patternfly/components/Card/card.scss#L401-L408